### PR TITLE
[MM-22433] Create cloudwatch alerts and alerting system for ELBs

### PR DESCRIPTION
**Pull request by cpanato**
#### Summary
This PR adds two lambda functions and terraform code to deploy a solution to create alerts and alerting when an internal LB from the K8s clusters goes down

We had some issues in the past that the LB was not responding because the services were out of service and we were blind and took a long time to discovery

This solution takes care of the entire lifecycle of the LB.
When the LB is created and matches the pattern which now is only for internal LB and the name starts with `api-`, which is the LB that KOPS create for the API Server, we create a Cloudwatch alert that will send the alerting message to an SNS queue.

Another Lambda function that is attached to the SNS topic will process this message and send a message to the Mattermost channel to the specific environment and if the ENV is prod or stg will also trigger an OpsGenie alert.

If the LB selfheal it also will close the opened Opsgenie alert.

If the LB is deleted the function will remove the Cloudwatch alert.

#### Ticket Link
Jira: https://mattermost.atlassian.net/browse/MM-22433

The private values will code to another PR in the private repo

### CODE REVIEW

File: terraform/aws/environments/staging/4-command-and-control/1-cluster/variables.tf [hunk 1]
```diff
@@ -113,7 +113,25 @@ variable "eks_ami_id" {
...
+variable "mattermost_hook" {
+  default     = ""
+  type        = "string"
+  description = "The mattermost webhook to post the messages"
+}
+
+variable "opsgenie_apikey" {
+  default     = ""
+  type        = "string"
+  description = "The opsgenei apikey"
```
Comment (angeloskyratzakos) [inline terraform/aws/environments/staging/4-command-and-control/1-cluster/variables.tf]: small typo on the opsgenei

Comment (cpanato) [reply terraform/aws/environments/staging/4-command-and-control/1-cluster/variables.tf]: yep, thanks! it is fixed :)

File: terraform/aws/environments/prod/4-command-and-control/1-cluster/variables.tf [hunk 2]
```diff
@@ -25,95 +25,113 @@ variable "auth_private_subnet_ids" {
...
+variable "mattermost_hook" {
+  default     = ""
+  type        = "string"
+  description = "The mattermost webhook to post the messages"
+}
+
+variable "opsgenie_apikey" {
+  default     = ""
+  type        = "string"
+  description = "The opsgenei apikey"
```
Comment (angeloskyratzakos) [inline terraform/aws/environments/prod/4-command-and-control/1-cluster/variables.tf]: small typo opsgenei

Comment (cpanato) [reply terraform/aws/environments/prod/4-command-and-control/1-cluster/variables.tf]: done

**Comment by cpanato** [comment]
@angeloskyratzakos @stylianosrigas deployed to all envs

**Pull request merged**